apps/verify/behaviours/email-lookup-sender.js

```diff
@@ -44,22 +43,29 @@ module.exports = superclass => class extends superclass {
     if (this.skipEmailVerification(email)) {
       return super.saveValues(req, res, next);
     }
+    return super.saveValues(req, res, async err => {
+      if (err) {
+        return next(err);
+      }
+      try {
+        const organisation = req.sessionModel.get('user-organisation');
+        const emailDomain = email.replace(/.*@/, '');
 
-    return super.saveValues(req, res, err => {
-      const organisation = req.sessionModel.get('user-organisation');
-      const emailDomain = email.replace(/.*@/, '');
+        const isRecognisedEmail = emailDomainCheck.isValidDomain(emailDomain);
 
-      const isRecognisedEmail = emailDomainCheck.isValidDomain(emailDomain);
+        if (!isRecognisedEmail) {
+          req.sessionModel.set('recognised-email', false);
+          return next(err);
+        }
 
-      if (!isRecognisedEmail) {
-        req.sessionModel.set('recognised-email', false);
+        const host = req.get('host');
+        const token = tokenGenerator.save(email, organisation);
```

Comment:
```suggestion
        const token = await tokenGenerator.save(email, organisation);
```
you'll have to add the keyword `async` above

Reply:
👍🏾 Have amended this now